.gitignore

```diff
@@ -2,4 +2,4 @@
 public
 node_modules
 .env
-.contentful/export-model-and-content.json
\ No newline at end of file
+.contentful/*.json
```

Comment:
@thinkybeast has some changes to the setup coming, doing something similar to the blog starter, but I don't think we want to ignore all JSON files in this directory, since one of them is the content export file that is used to populate a new space